src/Eventuras.WebApi/Controllers/Orders/OrdersController.cs

```diff
@@ -71,6 +90,42 @@ public async Task<IActionResult> ListAccessibleOrders([FromQuery] OrdersQueryDto
                 query, paging, o => new OrderDto(o)));
         }
 
+        [HttpPost]
+        public async Task<IActionResult> CreateOrUpdateOrder([FromBody] NewOrderRequestDto request)
+        {
+            if (!ModelState.IsValid)
+            {
+                return BadRequest(ModelState.FormatErrors());
+            }
+
+            try
+            {
+                var registration = await _registrationRetrievalService
+                    .GetRegistrationByIdAsync(request.RegistrationId);
+
+                await _registrationAccessControlService
+                    .CheckRegistrationUpdateAccessAsync(registration);
+
+                // FIXME: calling obsolete API here
```

Comment:
@andy-a-o When time - feel free to take some hours cleaning up the use of the obsolete API here later!